library/src/main/java/org/jellyfin/apiclient/model/dto/BaseItemPerson.java

```diff
@@ -58,15 +60,31 @@ public final void setRole(String value)
 	 <value>The type.</value>
 	*/
 	private String Type;
+	@Deprecated
 	public final String getType()
```

Comment:
Can you remove the deprecated methods or are they still used somewhere?

Reply:
We decided to leave them in because the Android (not TV) app uses the master branch and we don't want to cause issues over there.

It shouldn't actually cause issues as the Android app doesn't use these models but just to be sure :)

Reply:
Unclear, Android TV doesn't, other clients still might.